benchmarks/dynamo/common.py

```diff
@@ -2095,6 +2095,10 @@ def setup_amp(self, current_device=None):
                 )
                 self.autocast_arg["dtype"] = amp_dtype
 
+    def empty_gpu_cache(self):
+        torch.cuda.empty_cache()
+        torch.xpu.empty_cache()
+
```

Comment:
This function should be a utility function and it does not only serve `BenchmarkRunner`. Could we lift this function just like other utility functions of this file?

Reply:
Thanks for the review! Fixed in https://github.com/pytorch/pytorch/pull/126377/commits/65bacb7340cddbc8bbf95728bf98b703812d4e8b